### PR TITLE
[docs] add a header template for es exporter and receiver.

### LINKED ISSUE #10158: Add status header for elasticsearch

### LINKED ISSUE #10159: Add status header for elasticsearch

### PR DESCRIPTION
**Pull request by Frapschen**
**Description:** <Describe what has changed.>
hi, I add a header template for es exporter and receiver,pls review.

Fixes #10158
Fixes #10159

### CODE REVIEW

File: exporter/elasticsearchexporter/README.md [hunk 1]
```diff
@@ -1,5 +1,16 @@
 # Elasticsearch Exporter
 
+| Status                   |         |
+| ------------------------ |---------|
+| Stability                | beta    |
+| Supported pipeline types | logs    |
+| Distributions            | contrib |
```
Comment (TylerHelmuth) [inline exporter/elasticsearchexporter/README.md:7]:
```suggestion
| Status                   |           |
| ------------------------ |---------- |
| Stability                | [beta]    |
| Supported pipeline types | logs      |
| Distributions            | [contrib] |
```

File: exporter/elasticsearchexporter/README.md [hunk 2]
```diff
@@ -1,5 +1,16 @@
...
+
+<README contents>
+
+[beta]:https://github.com/open-telemetry/opentelemetry-collector-contrib#beta
```
Comment (TylerHelmuth) [inline exporter/elasticsearchexporter/README.md:11]:
```suggestion
[beta]:https://github.com/open-telemetry/opentelemetry-collector#beta
```

File: receiver/elasticsearchreceiver/README.md [hunk 3]
```diff
@@ -1,8 +1,19 @@
 # Elasticsearch Receiver
 
+
+| Status                   |          |
+| ------------------------ |----------|
+| Stability                | beta     |
+| Supported pipeline types | metrics  |
+| Distributions            | contrib  |
```
Comment (TylerHelmuth) [inline receiver/elasticsearchreceiver/README.md:8]:
```suggestion
| Status                   |            |
| ------------------------ |----------- |
| Stability                | [beta]    |
| Supported pipeline types | metrics    |
| Distributions            | [contrib]  |
```

File: receiver/elasticsearchreceiver/README.md [hunk 4]
```diff
@@ -1,8 +1,19 @@
...
+
+<README contents>
+
+[beta]:https://github.com/open-telemetry/opentelemetry-collector-contrib#beta
```
Comment (TylerHelmuth) [inline receiver/elasticsearchreceiver/README.md:12]:
```suggestion
[beta]:https://github.com/open-telemetry/opentelemetry-collector#beta
```

File: exporter/elasticsearchexporter/README.md [hunk 5]
```diff
@@ -1,5 +1,16 @@
 # Elasticsearch Exporter
 
+| Status                   |         |
+| ------------------------ |---------|
+| Stability                | beta    |
+| Supported pipeline types | logs    |
+| Distributions            | contrib |
+
+<README contents>
```
Comment (codeboten) [inline exporter/elasticsearchexporter/README.md:9]: this shouldn't be here

File: receiver/elasticsearchreceiver/README.md [hunk 6]
```diff
@@ -1,8 +1,19 @@
 # Elasticsearch Receiver
 
+
+| Status                   |          |
+| ------------------------ |----------|
+| Stability                | beta     |
+| Supported pipeline types | metrics  |
+| Distributions            | contrib  |
+
+<README contents>
```
Comment (codeboten) [inline receiver/elasticsearchreceiver/README.md:10]: this shouldn't be here

File: exporter/elasticsearchexporter/README.md [hunk 7]
```diff
@@ -1,5 +1,16 @@
...
+
+[beta]:https://github.com/open-telemetry/opentelemetry-collector-contrib#beta
+[contrib]:https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol-contrib
```
Comment (codeboten) [inline exporter/elasticsearchexporter/README.md:12]: please move the links to the bottom

**Comment by codeboten** [comment]
Pinging codeowners to validate the beta status: @urso @faec @blakerouse
@djaglowski @binaryfissiongames

File: receiver/elasticsearchreceiver/README.md [hunk 8]
```diff
@@ -1,8 +1,19 @@
...
+
+[beta]:https://github.com/open-telemetry/opentelemetry-collector-contrib#beta
+[contrib]:https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol-contrib
+
 This receiver queries the Elasticsearch [node stats](https://www.elastic.co/guide/en/elasticsearch/reference/current/cluster-nodes-stats.html) and [cluster health](https://www.elastic.co/guide/en/elasticsearch/reference/current/cluster-health.html) endpoints in order to scrape metrics from a running elasticsearch cluster.
 
-Supported pipeline types: `metrics`
 
 > :construction: This receiver is in **BETA**. Configuration fields and metric data model are subject to change.
```
Comment (BinaryFissionGames) [inline receiver/elasticsearchreceiver/README.md:18]: We can remove this disclaimer since the new header will call out that this receiver is in beta.

**Review by BinaryFissionGames — COMMENTED** [review]
General info here looks good on the receiver side, just one note in addition to @codeboten and @TylerHelmuth's suggestions.

**Comment by Frapschen** [comment]
probably everything is done., can merged?

**Review by djaglowski — APPROVED** [review]

**Pull request merged**